config/crds/gcp_v1beta1_gcpnamespacerestriction.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: gcpnamespacerestrictions.gcp.kiwigrid.com
+spec:
+  group: gcp.kiwigrid.com
+  names:
+    kind: GcpNamespaceRestriction
+    plural: gcpnamespacerestrictions
+  scope: Cluster
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            namespace:
+              type: string
+            regex:
```

Comment:
I would prefer to get rid of this `regex` flag and handle `namespaces` and `resources` always as regular expressions.
@monotek What are your thoughts on this? Do you agree or do you've any concerns about it?

Reply:
No concerns if "^" and "$" is always added to the beginning and end of the value automatically if not exists.